src/main/java/Database/Interaction/Presentation/Servlets/GENERIC_INFO.java

```diff
@@ -42,6 +42,8 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
             CustomLogs.InfoLog("Entered " + SERVLET_URL + ".", true);
 
             // Base
+            req.setCharacterEncoding("UTF-8");
```

Comment:
zakladny problem tvojho kodu z minuleho semestra je ze sa extremne vela sekvencii opakuje na roznych miestach. Zatial to vladzes manazovat ale pockaj ono si ta to pocka

Reply:
Takéto veci ako req.setCharacterEncoding a zdielane kody neviem ako rozumnejšie vyriešiť. Ak máš nápad daj vedieť. Rád by som to zlepšil, ale rozumným spôsobom, lebo to, že uzavriem kód do nejakej funkcie s tým, že si tam tie objekty iba vrátim mi nepríde až tak ok.